addons/blocks2image/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Save blocks as image",
-  "description": "Right click the editor workspace to export blocks as SVG/PNG images.",
+  "description": "Right click the code area to export blocks as SVG/PNG images.",
```

Comment:
We might also change this soon similarly to other code comments I made above.

Reply:
I'm not sure how exactly this addon will behave after #1754, but the ability to right click the code area to get an image of all scripts should not be removed.